lib/install/config/assets.js

```diff
@@ -0,0 +1,40 @@
+// Handles static asset integration - like sass and images
+// Note: You must restart bin/webpack-watcher for changes to take effect
+
+const ExtractTextPlugin = require('extract-text-webpack-plugin')
+const process = require('process')
+const sharedConfig = require('./shared.js')
+const { devServer } = require('../../package.json')
+
+const production = process.env.NODE_ENV === 'production'
+const hotServerAddr = `http://${devServer.host}:${devServer.port}/`
```

Comment:
> This is what I was referring when I said it was complicated to read this config out of ruby.

I think this is where we are disconnected. The gist of what I'm pointing out is that the pattern of passing the Rails configuration options over to the Webpack config already exists in the current code using environment variables. You don't need to read the Ruby config from JS. It is passed into the JS config using environment variables.